### DISCUSSION TITLE
Teams - enviroments

### DISCUSSION THREAD

**Discussion by davew94** [hoppscotch/hoppscotch#1712] [Ideas]
Within my business there are multiple development & support teams that have access to different enviroments. 

Like "Team collections" i want to have the ability to sync enviroments across a team rather than each user having their own "my" enviroments. 

The beneift of this would be that if new members that join the development or support teams they are automatically synced to have all the enviroment varaibles without having to set this up themselves.

**Comment by liyasthomas** [comment]
Awesome feature request. As soon as we roll out Teams collections to production - we'll add Teams environments too. Thank you for your patience. Stay tuned.

**Comment by anstosa** [comment]
Has this been implemented yet? I think not but I could be missing something

**Comment by bfgod-epicdevs** [comment]
Upvoting this, we have lot's of projects and every collection needs at least 3-4 envs, we are ending up with 40+ envs, not very convinient.
Also envs are not sync-ing?

If envs are part of the team/project and sync too, this would be totally amazing.

**Comment by liyasthomas** [comment]
Track the progress of this feature here: https://github.com/hoppscotch/hoppscotch/pull/2512

**Comment by AndrewBastin** [comment]
This feature is now implemented. Hence, closing this discussion.
